Source/WTF/Scripts/Preferences/WebPreferencesInternal.yaml

```diff
@@ -963,6 +963,20 @@ StreamingDeclarativeShadowDOMEnabled:
     WebCore:
       default: false
 
+SuspendBackgroundWebContent:
```

Comment:
The name is a little confusing, I would call this "BackgroundWebContentSuspensionEnabled". It will take the call site more readable IMO.

Reply:
Thanks, I changed it

Reply:
I  changed it again to BackgroundWebContentRunningBoardThrottlingEnabled since we don't actually suspend yet with this change.